tests/UI/pages/FO/myAccount/orderDetails.js

```diff
@@ -110,6 +110,24 @@ class OrderDetails extends FOBasePage {
   async clickOnReorderLink(page) {
     await this.clickAndWaitForNavigation(page, this.reorderLink);
   }
+
+  /**
+   * Get delivery address
+   * @param page {Page} Browser tab
+   * @returns {Promise<string>}
+   */
+  async getDeliveryAddress(page) {
+    return this.getTextContent(page, '#delivery-address');
```

Comment:
Could you move the selector to selectors list in constructor ?

Reply:
done thx :+1: